chrome/script.browserify.js

```diff
@@ -222,14 +298,16 @@ function keyHandler(e) {
       switchFocus("div.entry:first-child > .login", "nextElementSibling");
       break;
     case "c":
-      if (e.ctrlKey) {
+      if (e.target.id != "search-field" && e.ctrlKey) {
```

Comment:
both `copy` shortcuts seem to not work for me anymore, do they work for you?

Reply:
Investigating...

Reply:
These seem to be working for me. Perhaps fixed by one of the other commits?

Are you still having issues, or are they working for you now?

Reply:
Emm they actually don't even work for me in release version... okay, nevermind, I'll deal with them separately.

Reply:
One thought - is the item actually selected when you use the copy shortcuts? I did change the behavior so that they did not run when the search box was selected, because they conflicted with entering filter text (i.e. typing a capital C resulted in a copy-username-and-close behavior, which is obviously undesirable). That's what the `target-id` check is for.